tests/tests_tls_2019.yml

```diff
@@ -80,5 +81,39 @@
         __verify_mssql_password: "p@55w0rD"
         __verify_mssql_is_tls_encrypted: false
 
+    # Test tls_remote: true
+    - name: Remove certificates from hosts
+      file:
+        path: /etc/pki/tls/{{ item }}
+        state: absent
+      loop: "{{ __mssql_certs }}"
+
+    - name: Copy certificates to hosts
+      copy:
+        src: "{{ item }}"
+        dest: ~/{{ item }}
```

Comment:
using `~` in this context works?

Reply:
It did when I run CI locally with tox

Reply:
I should rather copy them to /tmp/{{ item }} though